packages/flutter_tools/lib/src/commands/format.dart

```diff
@@ -36,7 +56,24 @@ class FormatCommand extends FlutterCommand {
     }
 
     final String dartfmt = sdkBinaryName('dartfmt');
-    final List<String> cmd = <String>[dartfmt, '-w']..addAll(argResults.rest);
+    final List<String> cmd = <String>[dartfmt];
```

Comment:
please rename "cmd" so that it's not an abbreviation (see our style guide). Thanks.

Reply:
Thank you for your pointed out. I fixed it at 3f1a299f2bc757423149726005fd173014906438.